### DISCUSSION TITLE
[FF8] Use High-Res Models only

### DISCUSSION THREAD

**Discussion by Serie9** [julianxhokaxhiu/FFNx#561] [Ideas]
As shown in this thread here: https://forums.qhimm.com/index.php?topic=19979.0

It would be super cool if it were possible to implement that with FFNx.

Maybe a toggle that's like: "use_highres_models_only = true/false"

In the thread it mentioned (Reply#8) that on some parts there might occur texture issues.
https://forums.qhimm.com/index.php?topic=15668.msg221354#msg221354 (example of texture issues)

So it's probably not just an easy replace this with that kind of thing to implement.

**Comment by julianxhokaxhiu** [comment]
Please track #33 for progress around this topic. Thank you.
